src/pages/dashboard.rs

```diff
@@ -326,48 +319,17 @@ pub fn Dashboard() -> Element {
                 }
             }
             div { class: "dashboard__footer grid-footer",
-                div { class: "dashboard__footer__pagination",
-                    span { class: "dashboard__footer__paginator",
-                        {translate!(i18, "dashboard.footer.paginator", from: current_page(), to: (((communities_by_address.len() as f64 + 1f64) / SKIP as f64) as f64).ceil())}
-                    }
-                    div { class: "dashboard__footer__paginators",
-                        IconButton {
-                            class: "button--avatar",
-                            size: ElementSize::Small,
-                            body: rsx!(
-                                Icon {
-                                    icon: ArrowLeft,
-                                    height: 24,
-                                    width: 24,
-                                    fill: "var(--white)"
-                                }
-                            ),
-                            on_click: move |_| {
-                                let current = current_page();
-                                current_page.set(current - 1);
+                Paginator {
+                    to: {
+                        let mut div: u8 = (communities_by_address.len() + 1).saturating_div(SKIP).try_into().expect("Failed to convert paginator");
```

Comment:
If doing the usize everywhere thing you would save yourself the try_into/expect and to make things a bit more concise a small trick to avoid the block and comparison with zero can be to add `.max(1)` after the division, only when it's zero it will return 1.